static/js/modals.js

```diff
@@ -190,7 +190,7 @@ function getCheckboxItemsAsCSV(fieldset) {
 function getCustomFields(event) {
   var message = "";
 
-  document.querySelectorAll('fieldset').forEach(function (formField) {
+  document.querySelectorAll('fieldset:not(#about-you)').forEach(function (formField) {
```

Comment:
This feels oddly specific for such a general form generation script. Could we not find a more generic way to do this either in the JS or the form itself